### PR TITLE
Add delete and list features for owners and pets

### LINKED ISSUE #45: Add delete and list feature
As a veterinarian, I want to be able to delete pet and pet owner profiles as well as list the ones currently in the system

### PR DESCRIPTION
**Pull request by naythee169**
Fixes #45

### CODE REVIEW

File: src/main/java/seedu/address/model/pet/UniquePetList.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ public void add(Pet toAdd) {
         if (contains(toAdd)) {
             throw new DuplicatePetException();
         }
+        System.out.println(toAdd);
```
Comment (jloh02) [inline src/main/java/seedu/address/model/pet/UniquePetList.java:48]: Is this log intentionally left in?

**Review by jloh02 — CHANGES_REQUESTED** [review]
Other than the extra print, LGTM thanks!

File: src/main/java/seedu/address/model/pet/UniquePetList.java (hunk 1 above)
Comment (naythee169) [reply src/main/java/seedu/address/model/pet/UniquePetList.java:48]: forgot to take that out, will update

**Review by Runus01 — APPROVED** [review]
LGTM! Good idea to use an abstract delete command and good testing of the new commands created.

**Review by Runus01 — APPROVED** [review]
LGTM

**Pull request merged**